cs/src/core/Index/FasterLog/FasterLogIterator.cs

```diff
@@ -97,43 +100,45 @@ internal unsafe FasterLogScanIterator(FasterLog fasterLog, BlittableAllocator<Em
         /// Async enumerable for iterator
         /// </summary>
         /// <returns>Entry, entry length, entry address</returns>
-        public async IAsyncEnumerable<(byte[], int, long)> GetAsyncEnumerable([EnumeratorCancellation] CancellationToken token = default)
+        public async IAsyncEnumerable<(byte[], int, long, long)> GetAsyncEnumerable([EnumeratorCancellation] CancellationToken token = default)
```

Comment:
Breaking change.

Reply:
This was missed out of the original API, its needed for correct async iteration with truncation.